Langs/0415.lang

```diff
@@ -1,14 +1,17 @@
 [MetaData]
 Language=Polish - Poland
+RTL=false
 Translator=BxOxSxS, micwoj92, mLipok, kp1971
-Version=2.3.1.0
+Version=2.3.2.0
 [Font]
 Extra Large=24
 Large=18
 Medium=10
 Small=7.4
 Name=Arial
 [Strings]
+Enable enhanced accessibility?=Włączyć rozszerzony dostęp?
```

Comment:
Tutaj "enhanced accessibility" przetlumaczyles jako "rozszerzony dostęp?" a na dole jako "ulepszonej dostępności".

lepiej by bylo, gbyby bylo przetlumaczone w ten sam sposob

Reply:
I jeszcze dla formalnosci, co @mLipok, @kp1971 o tym sadza.

Reply:
Extended to rozszerzona
Enhanced to ulepszona